client/components/mma/shared/Card.tsx

```diff
@@ -12,6 +12,7 @@ Card.Header = (props: {
 	minHeightTablet?: boolean;
 }) => {
 	const headerCss = css`
+		position: relative;
```

Comment:
What did this do?

Reply:
The gift ribbon is absolutely positioned so this ensures it's relative to the header rather than the entire page.